plugins/woocommerce/changelog/pr-53167

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Add checks for the global variable in the template functions
```

Comment:
It's a bit of a nit, but I wonder if we could be a little more specific/communicate the value of this change (we're preventing a set of fatal errors) a little more clearly?

```suggestion
Add checks to various template functions, guard against fatal errors when the global $product object is unavailable.
```

This is just a thought—feel free to skip if I'm overthinking it.

Reply:
That's actually a nice improvement, so applying, thanks.